circe.el

```diff
@@ -2074,6 +2073,8 @@ SERVER-BUFFER is the server buffer of this chat buffer."
                    circe-network)))
     (setq lui-logging-format-arguments
           `(:target ,circe-chat-target :network ,network)))
+  (when (with-circe-server-buffer circe-logging)
```

Comment:
I'd move that into the preceding `let` for consistency.

Reply:
Sure thing. However, I can't reduce the noise by wrapping the `let` with `with-circe-server-buffer` (the `setq` must be done in the current buffer).

Reply:
Thanks, that's sufficient.